JitsiConference.js

```diff
@@ -1296,22 +1296,23 @@ JitsiConference.prototype.onIncomingCall
 
     // Add local tracks to the session
     try {
-        jingleSession.addLocalTracks(this.getLocalTracks()).then(() => {
-            jingleSession.acceptOffer(jingleOffer, null,
-                error => {
-                    GlobalOnErrorHandler.callErrorHandler(error);
-                    logger.error(
-                        'Failed to accept incoming Jingle session', error);
-                }
-            );
-
-            // Start callstats as soon as peerconnection is initialized,
-            // do not wait for XMPPEvents.PEERCONNECTION_READY, as it may never
-            // happen in case if user doesn't have or denied permission to
-            // both camera and microphone.
-            this.statistics.startCallStats(jingleSession);
-            this._startRemoteStats();
-        });
+        jingleSession.acceptOffer(
```

Comment:
just wondering, why is it we no longer add the local tracks here (and the similar one below)?